### ISSUE TITLE
NTR: CT of temporal bone: modiolus hypoplasia (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8932]
New term request CT of temporal bone: modiolus hypoplasia (elthat_100922053426): Deficiency of modiolus (central axis in cochlea)

**Comment by pnrobinson** [comment]
@MickeySegal
I think this is either synonymous or overlapping with [Incomplete partition of the cochlea type I HP:0011374](https://hpo.jax.org/app/browse/term/HP:0011374)? Is there a need for a separate term?

**Comment by MickeySegal** [comment]
Sounds the same, but both of need synonyms to make this clear.  Ours is from https://www.ncbi.nlm.nih.gov/books/NBK1467/ "high-resolution CT revealed that 100% had deficiency of the modiolus (i.e., the bony polyhedral structure centered on the cochlea was not apparent on a mid-modiolar section)"

**Comment by pnrobinson** [comment]
@MickeySegal 
I have added Modiolus hypoplasia as a RELATED SYNONYM.
